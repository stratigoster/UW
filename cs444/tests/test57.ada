PACKAGE foo IS
  a,b,c,d,e,f,g : integer := 10;
begin
  declare
    c : boolean := false;
    function foo(a:integer; b:float; c:boolean) return integer is
    begin
      c := a > a or else b=b;
    end foo;
    i : integer := foo(c=>true,a=>2,d=>3.0);
--                                  ^^^^^^^
-- no parameter named `d'
  begin
    for x in 1..5 loop 
      i := i + 1;
    end loop;
    null;
  end;
END foo;
