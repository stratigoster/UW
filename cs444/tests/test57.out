***ERROR***:10:45:	No compatible function found

------
Compilation error
------
